Resources/Catalog/FileCatalog.py

```diff
@@ -34,6 +34,15 @@ class FileCatalog( object ):
 
   write_methods += write_meta_methods
 
+  lfn_methods = ['exists', 'isLink', 'readLink', 'isFile', 'getFileMetadata', 'getReplicas',
```

Comment:
Not necessarily, I would like to keep those independent. lfn_methods are those that have LFN standard form dictionary as its first argument. I do not want to distinguish meta and other methods any more.